apps/api/src/app/events/usecases/digest-filter-steps/digest-filter-steps-regular.usecase.ts

```diff
@@ -29,6 +29,7 @@ export class DigestFilterStepsRegular {
         {
           $set: {
             _digestedNotificationId: delayedDigests._notificationId,
+            expireAt: delayedDigests.expireAt,
```

Comment:
If notification has been digested into an existing one, will expire at the same date as the original one.